src/libs/Fullstory/index.ts

```diff
@@ -130,9 +137,16 @@ const FS = {
         try {
             Environment.getEnvironment().then((envName: string) => {
                 const isTestEmail = value.email !== undefined && value.email.startsWith('fullstory') && value.email.endsWith(CONST.EMAIL.QA_DOMAIN);
-                if ((CONST.ENVIRONMENT.PRODUCTION !== envName && !isTestEmail) || isExpensifyTeam(value?.email)) {
+                if (
+                    (CONST.ENVIRONMENT.PRODUCTION !== envName && !isTestEmail) ||
+                    Str.extractEmailDomain(value.email ?? '') === CONST.EXPENSIFY_PARTNER_NAME ||
+                    Session.isSupportAuthToken()
+                ) {
+                    // On web, if we started FS at some point in a browser, it will run forever. So let's shut it down if we don't want it to run.
+                    FullStory('shutdown');
```

Comment:
NAB: the command names should probably go in CONST

Reply:
done